apps/deploy-web/src/queries/useExactDeploymentGrantsQuery.ts

```diff
@@ -0,0 +1,8 @@
+import { useQuery } from "react-query";
+
+import { useAllowanceService } from "@src/hooks/useAllowanceService";
+
+export function useExactDeploymentGrantsQuery(granter: string, grantee: string, { enabled = true } = {}) {
+  const allowanceHttpService = useAllowanceService();
+  return useQuery(["DeploymentGrant", granter, grantee], () => allowanceHttpService.getDeploymentGrantsForGranterAndGrantee(granter, grantee), { enabled });
```

Comment:
We have a file `queryKeys` for all the react-query keys, it would be nice to add them all there.

Reply:
changing. however I'd like to point out imo it's not the best approach. It makes every query depend on smth that every other query depends on. Basically if we want to use that query in another project we'll have to do some changes.

What is the purpose of this?

Reply:
The file is local to deploy-web and it makes it easier to follow the same naming and avoid key collision.

Reply:
on the other hand it would be easier if it was in every query file. collisions can be managed in a different way. e.g. we could have a registry to which every query added it's keys.

Reply:
Found this https://github.com/lukemorales/query-key-factory

Reply:
Anyways, something to consider in the future.